actors/miner/src/lib.rs

```diff
@@ -33,6 +33,7 @@ use fvm_shared::econ::TokenAmount;
 // The following errors are particular cases of illegal state.
 // They're not expected to ever happen, but if they do, distinguished codes can help us
 // diagnose the problem.
+pub use beneficiary::*;
```

Comment:
Move this line to the end of the block next to other `pub use`.

Reply:
seem rust-fmt not allow this case

Reply:
hmm, weird